src/utils/themes/dark_theme.ts

```diff
@@ -176,6 +176,11 @@ export const DARK_THEME: Theme = {
       visible: true,
     },
     line: {
+      stroke: '#535966',
```

Comment:
The `line.stroke` and `crossLine.stroke` could use the same color. And `#999` has more contract on darker backgrounds:

```suggestion
      stroke: '#999',
```

Reply:
done in #5e8f506